docker.sh

```
@@ -1,2 +1,2 @@
 sudo docker build -t letraz-server:Djnago .
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)